WordPress/gutenberg
# Image block: Fix focus style not around whole image when linked

## What?
Fixes: #62544 

## Why?
When the image block is linked the focus style for the link does not stretch around the whole image.

## How?
Adds display property to `Image` block and `Media & Text` block.

## Testing Instructions

1. Open a post or page.
2. Insert Image or media & Text block.
3. Link the Image with a link and open the post/page in frontend.
4. Use the tab button to go to that Image and see the focus style is wrapping to whole Image now.

## Screenshots or screencast 
https://github.com/WordPress/gutenberg/assets/77401999/25dbf467-38f8-4303-83dc-c2f6e83cbcea
